app/src/main/java/org/fossasia/openevent/app/data/models/Ticket.java

```diff
@@ -19,7 +20,7 @@
     private long quantity;
     private String type;
 
-    @ForeignKey(stubbedRelationship = true)
+    @ForeignKey(stubbedRelationship = true, onDelete = ForeignKeyAction.CASCADE)
```

Comment:
>Meaning, it is saying if the event foreign key is deleted (when ticket will be deleted), delete the event too

No it works in the other way. 
>Foreign key ON DELETE and ON UPDATE clauses are used to configure actions that take place when deleting rows from the parent table (ON DELETE), or modifying the parent key values of existing rows (ON UPDATE).

here parent is event
and child is ticket and social link.

So I think nothing wrong. What do you say?